### ISSUE TITLE
Tiup playground can't use `~` as directory

### ISSUE THREAD

**Issue by wjhuang2016** [pingcap/tiup#619]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

tiup playground --db.config=~/cp.toml

2. What did you expect to see?
Start success.

3. What did you see instead?
```
➜  tidb git:(master) tiup playground --db.config=~/cp.toml
Starting component `playground`:  --db.config=~/cp.toml
Use latest stable version: v4.0.2

	You can specified the version by running "tiup playground <version>", example: "tiup playground v4.0.0"

Playground Bootstrapping...
Start pd instance...
Start tikv instance...
Start tidb instance...
............................................................To view the dashboard: http://127.0.0.1:2379/dashboard
To view the Prometheus: http://127.0.0.1:9090
To view the Grafana: http://127.0.0.1:3000
^CGot signal interrupt (Component: playground ; PID: 80635)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1ae2512]

goroutine 22 [running]:
github.com/pingcap/tiup/components/playground/instance.(*TiFlashInstance).Pid(0xc0002bc820, 0x1e03e1c)
	<autogenerated>:1 +0x32
main.(*Playground).terminate.func1(0x1e03e1c, 0x7, 0x1fccfe0, 0xc0002bc820, 0x0, 0x0)
	github.com/pingcap/tiup@/components/playground/playground.go:898 +0x1f4
main.(*Playground).RWalkInstances(0xc0002ba000, 0xc000383f90, 0x8000, 0xc00025b790)
	github.com/pingcap/tiup@/components/playground/playground.go:493 +0x124
main.(*Playground).terminate(0xc0002ba000, 0x2, 0xc00037c000, 0x2, 0x2)
	github.com/pingcap/tiup@/components/playground/playground.go:891 +0x68
created by main.(*Playground).bootCluster.func3
	github.com/pingcap/tiup@/components/playground/playground.go:844 +0x1df
Component `playground` exit with error: exit status 2
```

4. What version of TiUP are you using (`tiup --version`)?
➜  tidb git:(master) tiup --version
v1.0.8 tiup
Go Version: go1.13
Git Branch: master
GitHash: 4276089

**Comment by wjhuang2016** [comment]
It seems to be a TiDB problem or Mac OS problem
![image](https://user-images.githubusercontent.com/17380469/88139714-dc961880-cc22-11ea-98c5-e33601a15515.png)

**Comment by lonng** [comment]
@wjhuang2016 Yes, tidb has the same problem, but I will fix it #620.
